.github/workflows/release.yml

```diff
@@ -429,32 +429,31 @@ jobs:
             org.opencontainers.image.description=Solarwinds OTEL distro Java agent
             org.opencontainers.image.vendor=SolarWinds Worldwide, LLC
 
-      - name: Build and push
-        uses: docker/build-push-action@v5
+      - name: Build and push -> linux/amd64
+        uses: docker/build-push-action@v6
         with:
           push: true
           context: agent
-          platforms: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
+          platforms: linux/amd64
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          load: true
 
-      - name: Analyze for critical and high CVEs
-        id: docker-scout-cves
+      - name: Analyze for critical and high CVEs -> linux/amd64
         uses: docker/scout-action@v1
         with:
           command: cves
-          image: "fs://./"
+          image: ${{ steps.meta.outputs.tags[0] }}
           platform: "linux/amd64"
-          ignore-base: true
-          only-package-types: maven
 
```

Comment:
i see, you just split out the amd64 build and scan it then build the rest of the platforms in the next step, nice.

do you want me to log a Jira for the finding from the test run's [scout scan](https://github.com/solarwinds/apm-java/actions/runs/13337031290#summary-37254419729) (which looks much better now) or on your radar already?

Reply:
let's log ticket